pure_changelog.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased
 
 ### Fixed
+* Fixed order services while closing the system
+* Fixed crash of the Eink service while restoring data
```

Comment:
what data? :P if OS, maybe it would be better to write it explicit ;)

Reply:
Ok, you're right :)

Reply:
Done :)